languages/python/default.nix

```diff
@@ -49,6 +49,79 @@ in {
       defaultText = lib.literalExpression "pkgs: [ ]";
       example = lib.literalExpression "pkgs: [ pkgs.requests ]";
     };
+    config = lib.mkOption {
+      type = with lib.types; nullOr lines;
+      description = ''
+        Python interpreter startup configuration. See
+        <https://docs.python.org/3/using/cmdline.html#envvar-PYTHONSTARTUP>
+        for details.
+      '';
+      default = null;
+      defaultText = lib.literalExpression "null";
+      example = lib.literalExpression ''
+        import numpy as np
+      '';
+    };
+    configPath = lib.mkOption {
+      type = lib.types.path;
+      description = ''
+        Python interpreter startup configuration file path. See
+        <https://docs.python.org/3/using/cmdline.html#envvar-PYTHONSTARTUP>
+        for details.
+      '';
+      default = "${config.xdg.configHome}/python/startup.py";
+      defaultText = "$XDG_CONFIG_HOME/python/startup.py";
```

Comment:
It’s weird to use `$` interpolation in the example text when you actually use Nix antiquotation based on the option value, since `config.sessionVariables` _would_ interpolate that automatically, but only if `$XDG_CONFIG_HOME` is defined.

Reply:
Maybe leave this `null` by default.